example/Gruntconfig.json

```diff
@@ -1,12 +1,15 @@
 {
   "srcPaths": {
-    "make": "src/project.make",
+    "make": "src/*.make",
```

Comment:
I think this needs to stay as `src/project.make`. We should add `src/*.make` to the `projFiles` example so those get packaged up.